dom.bs

```diff
@@ -4502,6 +4502,8 @@ dom-Range-extractContents, dom-Range-cloneContents -->
   <a for=ShadowRoot>clonable</a> is true:
 
   <ol>
+   <li><p>Assert: <var>copy</var> is not a <a for=Element>shadow host</a>.
```

Comment:
(It's weird that this has for=Element, but that's a pre-existing issue.)

Reply:
Want me to change something?

Reply:
If you want to take it on, it should be a separate PR making it a standalone term that doesn't belong to Element.